src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -137,32 +152,64 @@ class CrashHeader extends React.Component {
           >
             {t('Raw')}
           </a>
-        </div>
-        <div className="btn-group">
-          {this.hasMinified() && [
-            <a
-              key="original"
-              className={
-                (stackType === 'original' ? 'active' : '') + ' btn btn-default btn-sm'
-              }
-              onClick={() => this.setStackType('original')}
-            >
-              {this.getOriginalButtonLabel()}
-            </a>,
-            <a
-              key="minified"
-              className={
-                (stackType === 'minified' ? 'active' : '') + ' btn btn-default btn-sm'
-              }
-              onClick={() => this.setStackType('minified')}
-            >
-              {this.getMinifiedButtonLabel()}
-            </a>,
-          ]}
-        </div>
-      </div>
+        </ButtonGroup>
+        {this.hasMinified() && (
+          <ButtonGroup className="btn-group">
+            {[
+              <a
+                key="original"
+                className={
+                  (stackType === 'original' ? 'active' : '') + ' btn btn-default btn-sm'
+                }
+                onClick={() => this.setStackType('original')}
+              >
+                {this.getOriginalButtonLabel()}
+              </a>,
```

Comment:
You could use the `Button` element here to remove more LESS classname usage.

Reply:
Sure, but then I need to add some styles. If there's no problem, I'd rather do it when I refactor the entire component

Reply:
Sure..but I'd prefer to do it  when I refactor the entire component

Reply:
I don't think we can use `<Button>` because it's apart of a `btn-group`